codecov.yml

```diff
@@ -1,7 +1,12 @@
 coverage:
   status:
     project:
+      # no leniency on `main`
+      main:
+        branches:
+          - main
+          - gh-readonly-queue/main/*
```

Comment:
@ericphanson Add a comment explaining that this is for GitHub Merge Queue?

Reply:
```suggestion
          # for GitHub merge queue
          - gh-readonly-queue/main/*
```